labelbox/data/serialization/ndjson/base.py

```diff
@@ -27,6 +29,15 @@ class NDJsonBase(BaseModel):
     def set_id(cls, v):
         return v or str(uuid4())
 
+    def dict(self, *args, **kwargs):
```

Comment:
This was added, so that when we generate the NDJson from annotation type and we call the dict() function, we pop either dataRow id or globalKey, whichever is not present. This is to prevent that in the case when id or globalKey are None, they aren't included in the final NDJson with "id: None" or "globalKey: None"